source/brailleDisplayDrivers/papenmeier_serial.py

```diff
@@ -28,31 +27,33 @@
 KEY_CHECK_INTERVAL = 10
 TIMEOUT = 0.5
 
-def brl_auto_id(): 
+def brl_auto_id() -> bytes:
 	"""send auto id command to braille display"""
-	return chr(STX)+'S'+chr(0)+chr(0)+chr(0)+chr(0)+chr(ETX)
 	#send a bad packet to the braille display
-	
-def brl_out(offset, data):
+	return bytes([STX, ord('S'), 0x0, 0x0, 0x0, 0x0, ETX])
+
+def brl_out(offset: int, data: List[int]) -> bytes:
 	"""send data to braille display"""
-	ret = []
-	ret.append(struct.pack('BB', STX, 0x53)) #STX,'S'
 	d2 = len(data)+7
-	ret.append(struct.pack('BB', offset / 256, offset % 256))
-	ret.append(struct.pack('BB', 0, d2 % 256))
-	for d in data:
-		ret.append(struct.pack('B', d))
-	ret.append(struct.pack('B', ETX))
-	return "".join(ret)
+	ret = bytearray([
+		STX,
+		ord('S'),
+		offset // 256,

```

Comment:
Wow, this really looks like poorman's bit shifting

```suggestion
		offset >> 8,
```

We can also leave it as is, though, but I consider it pretty missleading.
Note that I also ensured the validity of this change:

```
for i in range(0x1000000):
	assert i // 256 == i >> 8
```